ocp_resources/datavolume.py

```diff
@@ -272,26 +277,64 @@ def _check_none_pending_status(self, failure_timeout=120):
             self.logger.error(f"{self.name} status is {sample}")
             raise
 
-    def wait_for_dv_success(self, timeout=600, failure_timeout=120):
+    def wait_for_dv_success(
+        self,
+        timeout=TIMEOUT_10MINUTES,
+        failure_timeout=TIMEOUT_2MINUTES,
+        stop_status=None,
+        *args,
+        **kwargs,
+    ):
+        """
+        Wait until DataVolume succeeded with or without DV Garbage Collection enabled
+
+        Args:
+            timeout (int):  Time to wait for the DataVolume to succeed.
+            failure_timeout (int): Time to wait for the DataVolume to have not Pending/None status
+            stop_status (function): function that is called inside the TimeoutSampler
```

Comment:
Could we please rename `stop_status` to be more intuitive as it's a function?
Examples:

stop/break`_sample_function`
sample/check`_block_function`
`stop_status_func` etc...

Reply:
Done